### DISCUSSION TITLE
with github

### DISCUSSION THREAD

**Discussion by Kirtishukla2004** [community/community#30232] [Pull Requests] [answered]
hey 
as i am new to github i am not able to do a pull request and even not able to understand 

1. what is a "pull request"?
2. what's the diffrence  between pull request and an issue ?

**Comment by airtower-luna** [comment, marked as answer]
> what is a "pull request"?

You can create a pull request (PR) after making some changes on your own copy of a repository. Essentially making a PR tells the owner of the repository "I have some changes to your code, and I'd like you to accept them". Accepting those changes is called *merging* the PR. Of course you can use that to manage your own changes to your own repositories, too.

> what's the diffrence between pull request and an issue ?

An issue doesn't contain any code changes, it's to *communicate* a bug, feature request, or any other kind of *issue*. :wink: If solving an issue requires changes to the code it's common that someone will create a PR with the necessary code.

**Reply by Kirtishukla2004** [reply]
thanks for answering my issue

**Reply by airtower-luna** [reply]
Glad I could help! Marking the answer would be appreciated. :smile_cat:
